common/src/main/scala/org/apache/spark/sql/comet/util/Utils.scala

```diff
@@ -278,7 +277,7 @@ object Utils {
       case v @ (_: BitVector | _: TinyIntVector | _: SmallIntVector | _: IntVector |
           _: BigIntVector | _: Float4Vector | _: Float8Vector | _: VarCharVector |
           _: DecimalVector | _: DateDayVector | _: TimeStampMicroTZVector | _: VarBinaryVector |
-          _: FixedSizeBinaryVector | _: TimeStampMicroVector | _: StructVector) =>
+          _: FixedSizeBinaryVector | _: TimeStampMicroVector | _: StructVector | _: ListVector) =>
```

Comment:
This is the fix for https://github.com/apache/datafusion-comet/issues/1289

Reply:
Nice, need to include MapType as part of https://github.com/apache/datafusion-comet/pull/1603

Reply:
We could add `MapVector` here as well without any ill effects.